GTBitsOfGood/southface
# Add and edit notes in report builder

## PR Title/Tagline

Issue Number(s): #197 

- Replicate notes from card to report on add
- Editing notes in report builder will edit the replicated notes and not the original notes

### Checklist

- [ ] User can add additional notes in report builder
- [ ] Adding/Editing notes in the report builder should NOT propagate to the the digital library and affect the original notes of the standard card
- [ ] Any edits made to notes in report builder should successfully reflect on completed report

### How to Test

- Find card with notes
- Add card to report
- Verify that the original notes appear
- Modify notes on original card
- Verify that notes do not change on report
- Modify notes on card in report
- Verify that notes do not change on original card
- Add note in report builder
- Verify that new note does not appear for original card
